### PR TITLE
Introduce new icon to site.

### PR DESCRIPTION
**Pull request by etefera**
Addresses https://github.com/GoogleContainerTools/kpt/issues/1717 for kpt.dev
image
image
image

### CODE REVIEW

**Comment by etefera** [comment]
> Cool! Where did this icon come from? Did you design it?

@frankfarzan provided the image; I merely converted it to a vector graphic : )

**Comment by frankfarzan** [comment]
Very nice. A follow up (M3): The two shades of blue in the nav bar clash. We can either:
1. Change the navbar colors to match the logo.
2. Swap the colors in the logo on the navbar (blue on white)

Not a blocker.

File: site/index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="robots" content="noindex" />
-    <title>kpt - Home</title>
+    <title>Home</title>
```
Comment (frankfarzan) [inline site/index.html:6]: for seo, you probably want to have "kpt" in the title, not sure
I understand the repetition is annoying

**Review by frankfarzan — APPROVED** [review]

File: site/index.html (hunk 1 above)
Comment (etefera) [reply site/index.html:6]: Let's put it back in for now and we'll see if there's any feedback on it.

**Pull request merged**